final_project/dags/sql/enrich_user_profiles.sql

```diff
@@ -0,0 +1,36 @@
+DELETE FROM `{{ params.project_id }}.gold.user_profiles_enriched`
+WHERE TRUE
+;
```

Comment:
The whole table is deleted here, so users will not have access to it at the moment. This may lead to issues.